src/v2/models/AppState.js

```diff
@@ -185,6 +185,8 @@ export default Model.extend({
 
     if (identicalResponse && !isSameRefreshInterval) {
       this.set('dynamicRefreshInterval', this.getRefreshInterval(transformedResponse));
+    } else if (identicalResponse) {
+      this.trigger('change:dynamicRefreshInterval');
```

Comment:
so in case of identical response we always trigger `re-rendering` ? this might create a flicker type effect in `beacon` image..

Reply:
also you don't need else if.. we are triggering poll every time if response is identical

Reply:
Updated